### PR TITLE
Feat/US34437: Add NFT serial id to various flows and assets tab

### PR DESCRIPTION
**Pull request by anshulkhare-lab49**
**Description**:
Implemented Deposit Flow for NFTs
Screenshot 2023-08-30 at 12 18 09 AM

Screenshot 2023-08-30 at 12 18 00 AM

Screenshot 2023-08-30 at 12 17 55 AM
Screenshot 2023-08-30 at 12 17 33 AM

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: src/shared/ui-kit/components/FormTokenInput/FormTokenInputSelectTokenModal.tsx [hunk 1]
```diff
@@ -115,13 +118,13 @@ export function FormTokenInputSelectTokenModal() {
...
             <Flex alignItems="center" justifyContent="center" width="100%" height="100%" maxHeight="296px">
               <Text textStyle="p medium semibold">No results found.</Text>
             </Flex>
           ) : (
             <List variant="select-token-list">
-              <FixedSizeList height={296} itemCount={tokens.length} itemSize={64} width="100%">
+              <FixedSizeList height={296} itemCount={nonNFTTokens.length} itemSize={64} width="100%">
                 {({ index }) => <TokenListItem key={index} index={index} />}
               </FixedSizeList>
             </List>
```
Comment (roshan-bisht-lab49) [inline src/shared/ui-kit/components/FormTokenInput/FormTokenInputSelectTokenModal.tsx:130]: Screenshot 2023-08-31 at 11 18 58 AM
we could show some error that "do not have sufficient balance"

File: src/shared/ui-kit/components/FormTokenInput/FormTokenInputSelectTokenModal.tsx [hunk 2]
```diff
@@ -34,10 +34,11 @@ export function FormTokenInputSelectTokenModal() {
   const { value: searchTokensInput, handleChange: handleSearchTokensInputChanged } = useInput("");
   const tokensQueryResults = useAccountTokenBalances(assetListAccountId, { textSearch: searchTokensInput });
   const { data: tokens = [] } = tokensQueryResults;
+  const nonNFTTokens = tokens.filter((token) => !token.isNFT);
```
Comment (roshan-bisht-lab49) [inline src/shared/ui-kit/components/FormTokenInput/FormTokenInputSelectTokenModal.tsx:38]: could be `fungibleTokens`

File: src/dao/pages/MultiSigDAODashboard/DepositTokensModal.tsx [hunk 3]
```diff
@@ -34,28 +45,84 @@ export function DepositTokensModal(props: DepositTokensModalBodyProps) {
...
+          </Tabs>
+        )}
+        {tabIndex === 0 ? (
+          <>
+            <FormTokenInput
+              amountFormId="amount"
+              tokenFormId="tokenId"
+              assetListAccountId={safeId}
+              balanceAccountId={walletId ?? ""}
+              currentAmount={getValues().amount ?? ""}
```
Comment (roshan-bisht-lab49) [inline src/dao/pages/MultiSigDAODashboard/DepositTokensModal.tsx:65]: better to use destructured value `const { amount = "" } = getValues()` for all values used.

Comment (anshulkhare-lab49) [reply src/dao/pages/MultiSigDAODashboard/DepositTokensModal.tsx:65]: Done.

File: src/shared/ui-kit/components/FormTokenInput/FormTokenInputSelectTokenModal.tsx (hunk 2 above)
Comment (anshulkhare-lab49) [reply src/shared/ui-kit/components/FormTokenInput/FormTokenInputSelectTokenModal.tsx:38]: Changed the variable name

File: src/shared/ui-kit/components/FormTokenInput/FormTokenInputSelectTokenModal.tsx (hunk 1 above)
Comment (anshulkhare-lab49) [reply src/shared/ui-kit/components/FormTokenInput/FormTokenInputSelectTokenModal.tsx:130]: Done.

**Review by roshan-bisht-lab49 — APPROVED** [review]

**Pull request merged**